types/google-apps-script/google-apps-script.card-service.d.ts

```diff
@@ -9,6 +9,8 @@
 
 declare namespace GoogleAppsScript {
   namespace Card_Service {
+    import ConferenceData = GoogleAppsScript.Calendar.Schema.ConferenceData;
```

Comment:
avoid `import` statements in `.d.ts` files.

insert between lines 7 & 8 the following triple-slash directive:

```typescript
/// <reference path="google-apps-script.conference-data.d.ts" />
```